### ISSUE TITLE
Unable to compile: sunMoon.h:10: undefined reference to sunMoon::init(int, float, float)

### ISSUE THREAD

**Issue by maps2002** [patience4711/read-APSystems-YC600-QS1#15]
Hi there,

Thank you for the effort to put this together!

I am trying to compile the sketch from the source code, but it seems that the sunMoon implementation is missing:
`sunMoon.h:10: undefined reference to sunMoon::init(int, float, float)`

Is a `sunMoon.cpp` file missing in the source code, or am I doing something incorrect?

Thanks in advance.

**Comment by patience4711** [comment]
Maybe you should install that library. Can't you install the new version 8? I would like to know if it works as it should. 
Thanks

**Comment by maps2002** [comment]
Thanks. I didn't found the library thru Arduino IDE, but after Googling (should have done that since the start), I found it, and installed it. I can already build the sketch, so for the purpose if this issue I think it is solved.

I started with version 8, but I am having some kind of wifi connect issues, where the ESP connects to my wifi, but after a cold start it does not connect anymore (blue led always on).

I am going to try and debug it :|
